### DISCUSSION TITLE
Result of Microsoft RAG Hack

### DISCUSSION THREAD

**Discussion by Yash-Kavaiya** [microsoft/RAG_Hack#170] [General]
When will the official results of the Microsoft RAG Hack hackathon be published?

**Comment by multispark** [comment]
Hello Yash, we plan to have results posted around 9/27 or later!

**Reply by dheerajrhegde** [reply]
Hello @multispark - How would we know if our submission are being reviewed and if there are any questions that need to be clarified? Will we get an opportunity to present and highlight our thoughts or the decisions we took?

**Reply by multispark** [reply]
We are hard at work reviewing projects! Some judges have reached out on the issues submitted. We are not currently planning to have any more sessions to highlight projects. However, we will be making some blog posts/social posts about the winning projects of each category.

**Reply by dheerajrhegde** [reply]
Does it mean that we have not won if we have not been contacted for clarifications, information etc.?

**Reply by multispark** [reply]
Judges reached out if there were any missing resources (video, endpoint, etc.). This is not an indication of winning.

**Reply by erincon01** [reply]
Nobody contacted us either.
Neither to start up the openai and the azure postgres database. 
Seems like they didn't get interest in our football analysis scenario 🤔

**Reply by pamelafox** [reply]
Deployed endpoints were not required, judges were able to judge based off a codebases and videos. We'll be announcing winners soon!

**Reply by vikram-cloud-ai** [reply]
when do the results get published?

**Comment by dheerajrhegde** [comment]
Happy Monday @pamelafox  and @erincon01 - Reaching out to see of the results were published. Did not see any communication on the event page or on LinkedIn.

**Reply by jeremiah-louis** [reply]
Nothing has been released yet. I think when they publish it they'll let us know. Just exercise some patience. I don't think they have any motive of hoarding the results. They are probably wrapping up the process as we speak.

**Comment by erincon01** [comment]
I agree. We were 80+ projects. We all enjoyed the challenge, and we learned
a lot, that I believe was the main goal. Prices are secondary. You all know
the effort you have put into this!! Congrats to all!! 🔥💪

**Comment by multispark** [comment]
Hello everyone, thank you for your patience. We will be announcing winners on 10/7!
